nix/offchain.nix

```diff
@@ -318,6 +330,69 @@ in
           overlays = defaultCtlOverlays ++ additionalOverlays;
         };
 
+        # NOTE(Emily, 13 Jan 2023): This is currently semi-vendored from CTL. This shouldn't be necessary
+        # once we are on a more recent version that supports including the spago result in the bundle.
+        # 
+        # The exact difference here is what is specified by 'includeBundledModule'. Additionally,
+        # some work has been done to ensure this can work while being outside of the context of the
+        # 'project' scope (by taking 'project' as an argument).
+        #
+        # This workaround will no longer be necessary as soon as the PR is merged and we are up to date:
+        # https://github.com/Plutonomicon/cardano-transaction-lib/pull/1396
```

Comment:
This PR has since been merged, but we are not up to date yet, so this will have to wait for later.